### PR TITLE
spike/SC-1517

### PR DESCRIPTION
**Pull request by abnepc**
## Jira Story
[SC-1517](https://epcpower.atlassian.net/browse/SC-1517)
## About
Investigate/remove `sunspecdemo` from the repo
## Associated Pull Requests
* [stlib](https://github.com/epcpower/stlib/pull/48)
* [st](https://github.com/epcpower/st/pull/49)
* [grid-tied](https://github.com/epcpower/grid-tied/pull/683)
* [rm-dc](https://github.com/epcpower/rm-dc/pull/21)
* [m-pm](https://github.com/epcpower/m-pm/pull/4)
* [m-bcu](https://github.com/epcpower/m-bcu/pull/7)
* [m-tcu](https://github.com/epcpower/m-tcu/pull/53)
* [pm (rm-dc)](https://github.com/epcpower/pm/pull/94)
## Reproduction Steps
N/A
## Validation
Delete your `.venv` folder and do a `poetry install`. Then make sure that `sunspecdemo` was not installed. 

[SC-1517]: https://epcpower.atlassian.net/browse/SC-1517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -59,6 +52,7 @@ tqdm = "4.32.1"
 epyqlib = {path = "./sub/epyqlib",  develop = true}
 gitignoreio = "0.1.5"
 pyinstaller = "4.4"
+pyinstaller-hooks-contrib = "2022.10"
```
Comment (epcAnker) [inline pyproject.toml]: Note for this: when a `poetry update` was ran on develop, it failed due to `pyinstaller-hooks-contrib` updating.  Locking it to 2022.10 worked.

File: .vscode/settings.json
Comment (gordon-epc) [inline .vscode/settings.json:1]: Add a gitignore entry that ignores this file.

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (gordon-epc) [reply pyproject.toml]: Probably should add a comment saying this. It's possible we will need to adjust or remove in the future.

File: .vscode/settings.json
Comment (abnepc) [reply .vscode/settings.json:1]: [216e13d](https://github.com/epcpower/pm/pull/93/commits/216e13db8bd4c0d07bf9b8dd3dd779332a8c9bd8) & [0ed4637](https://github.com/epcpower/pm/pull/93/commits/0ed4637e139620f2ace87b52af131485d7dccf07) (renamed a commit made from GitHub)

File: pyproject.toml (hunk 1 above)
Comment (abnepc) [reply pyproject.toml]: [bcc867d](https://github.com/epcpower/pm/pull/93/commits/bcc867da3780770f3e10ccc067403132e91f059f)

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**